vulcanai2/plotters/visualization.py

```diff
@@ -21,350 +20,74 @@
 from sklearn.decomposition import PCA
 
 import matplotlib
-
-if os.name is not "posix":
-    if "DISPLAY" not in os.environ:
-        matplotlib.use('Agg')
-else:
-    matplotlib.use('TkAgg')
 import matplotlib.pyplot as plt
 
 import logging
 logger = logging.getLogger(__name__)
 
 
-#TODO: update to work with pytorch
```

Comment:
Why did you delete all this stuff? it will need to be implemented. did you move it elsewhere?

Reply:
We just need to add plotting stuff here. I was working on saliency map got stuck on that the whole day. since I have little knowledge on that. Some functions (`round_list`, `get_confusion_matrix`) are already moved to metrics, some like `get_timestamp` are not necessary in visualization, unless if you'd like them to have?

Reply:
the most important ones from there are all the functions starting with display (e.g. display_tsne, _plot_reduction, display_pca, display_saliency_map) and stuff like that

Reply:
resolved! (partially I guess)